2018-03-22T13:33:33.541Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)
    at /home/chris/workspace/logistics-chain/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-22T13:33:33.542Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/chris/workspace/logistics-chain/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-03-22T13:33:33.585Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)
    at /home/chris/workspace/logistics-chain/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-22T13:33:33.585Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'FCJPNGEN_NIS' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/chris/workspace/logistics-chain/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
